main.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
All code files (and many other script and config files) should start with a license header:

// Copyright 2020 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

To see an example of how this is used throughout a repo, see https://github.com/cloudspannerecosystem/harbourbridge.

Reply:
@ankitmalikg2

Reply:
@nevinheintze and @rosspatil I have added the license header.